ncat/docs/ncat.xml

```diff
@@ -942,6 +942,104 @@
     file.</para>
   </refsect1>
 
+  <reftsec1 id="ncat-diff">
+    <title> Differences between Ncat and Classical nc </title>
+    <para> There are a few changes and additional features in ncat as
+    compared to classical netcat. The following list highlights them
+    </para>
+
+    <variablelist>
+      <varlistentry>
+        <term>
+        <option>-i</option>
+        </term>
+        <listitem>
+        <para><option>-i</option> in netcat sets the delay interval for lines sent whereas
+        -i in ncat stands for idle read/write timeout</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>
+        <option>-w</option>
+        </term>
+        <listitem>
+        <para><option>-w</option> in netcat sets the timeout for connects and final net reads whereas
+            <option>-w</option> in ncat sets the connect timeout</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>
+        <option>-c</option>
+        </term>
+        <listitem>
+        <para><option>-c</option>  in netcat sends CRLF as line ending whereas
+            <option>-C</option>  does the same in ncat and <option>-c</option> in ncat executes the given command via /bin/sh </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>
+        <option>-e</option>
+        </term>
+        <listitem>
+        <para><option>-e</option> executes the given command in ncat</para>
```

Comment:
This option is the same in traditional and openbsd netcat, but may be missing from your distribution's package since it can be used to create a backdoor shell. Just leave it out of this section.